EFCore.VisualBasic.Templates/EFCore.VisualBasic.Templates.vbproj

```diff
@@ -17,7 +17,7 @@
     <PackageLicenseExpression>Unlicense</PackageLicenseExpression>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <Version>7.0.0</Version>
-    <Copyright>© 2022 Brice Lambson, et al. All rights reserved.</Copyright>
+    <Copyright>© 2023 Brice Lambson, et al. All rights reserved.</Copyright>
```

Comment:
@lorcQc Feel free to put your name here. lol, you can even remove mine--I'm happy just be part of the *et al.* at this point. 😉